core/driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/git/GitHandler.scala

```diff
@@ -75,20 +77,20 @@ object GitHandler {
     private val completion: AtomicInteger = new AtomicInteger()
     private val task: AtomicReference[String] = new AtomicReference()
 
+    private val titlePercentArray: mutable.ArrayBuffer[(String, Int)] = mutable.ArrayBuffer.empty
+
     @inline
-    private def renderProgress(completed: Int): String = {
+    private def printProgress(completed: Int): Unit = {
       val total = totalWork.get()
       val title = task.get()
-      val inner =
-        if (total == 0)
-          ""
-        else if (completed == total)
-          s"$title 100% ($total/$total)\n"
-        else {
-          val percent = (100 * completed) / total
-          s"$title $percent% ($completed/$total)"
-        }
-      s"\r$inner"
+
+      if (total == 0) return
+
+      val percent = (100 * completed) / total
+      if (!titlePercentArray.contains((title, percent))) { // to print only one line per 1 percent in logs
+        titlePercentArray += (title -> percent)
+        println(s"$title $percent% ($completed/$total)")
```

Comment:
what happened to print() and  \r?

Reply:
Huh `\r` was supposed to return to the beginning of the line, so that the previously written string could be overwritten right? 🤔 this doesn't work in most CIs, tho 🤔

Reply:
Yet it looks much better in terminal. The title says "Make git clone logging less verbose", while from perspective of terminal users it is more verbose as instead of replacing output it adds new for each %. Can we detect if output is connected to a tty (is it on ci?) or add some CI mode?

Reply:
+1, it probably can be just as easy as checking for `CI` env var

Reply:
Ok, @lukaszwawrzyk - I just fixed it.
- for local execution (terminal users) - logging will work exactly the same as before
- for github actions workflows - logging will use new logic (1 line per 1%)